src/hooks/useProcessorsApi/useListProcessorsApi.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Configuration,
+  InlineResponse2021,
+  ProcessorsApi,
+} from "@openapi/generated";
+import { useState } from "react";
+
+export function useListProcessorsApi(
```

Comment:
You have called the similar hook for bridges `useGetBridgesApi`. Could we use a similar naming, like `useGetProcessorsApi`?